python/src/file.rs

```diff
@@ -113,6 +113,33 @@ impl LanceColumnMetadata {
     }
 }
 
+#[pyclass(get_all)]
+#[derive(Clone, Debug, Serialize)]
+pub struct LanceFileStatistics {
+    columns: Vec<LanceColumnStatistics>,
```

Comment:
Could you write and unit test a `repr` method on these? That will make them much easier to use when debugging.

Reply:
added, thanks for the advice!